packs/rust/charts/values.yaml

```diff
@@ -0,0 +1,20 @@
+# Default values for Swift.
```

Comment:
Perhaps better to replace here swift with rust. To avoid confusions.

Reply:
Nice catch! Fixed :)